### PR TITLE
feat(ingest): add browse paths + dataplatform for Feast features

### PR DESCRIPTION
**Pull request by kevinhu**
(browse paths for SageMaker features have been added to #2830 as well)

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by gabe-lyons — APPROVED** [review]

File: metadata-ingestion/tests/integration/hive/hive_mces_golden.json [hunk 1]
```diff
@@ -1,409 +1,409 @@
...
-              {
-                "fieldPath": "foo",
-                "jsonPath": null,
-                "nullable": true,
-                "description": null,
-                "type": {
-                  "type": {
-                    "com.linkedin.pegasus2avro.schema.NumberType": {}
-                  }
+    "auditHeader": null,
```
Comment (hsheth2) [inline metadata-ingestion/tests/integration/hive/hive_mces_golden.json:3]: spurious diff?

Comment (kevinhu) [reply metadata-ingestion/tests/integration/hive/hive_mces_golden.json]: Fixed!

**Comment by shirshanka** [comment]
@kevinhu : this PR likely needs a rebase

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**